### ISSUE TITLE
Links in single quotes

### ISSUE THREAD

**Issue by joshiste** [SoapBox/linkifyjs#164]
When a link is surrounded by single quotes the link (text and href) contains the last single quote.

```
Linkify the following URL: 'https://github.com/SoapBox/linkifyjs/'
```

results in
```html
Linkify the following URL: '<a href="http://soapbox.github.io/linkifyjs/docs/options.html'" class="linkified" target="_blank">http://soapbox.github.io/linkifyjs/docs/options.html'</a>
```

**Comment by nfrasser** [comment]
Thanks for the bug report @joshiste, this has been fixed in [v2.1.2](https://github.com/SoapBox/linkifyjs/releases/tag/v2.1.2)!

**Comment by joshiste** [comment]
Holy moly! That was fast! Thanks a lot!
